script/hassfest/translations.py

```diff
@@ -128,9 +128,11 @@ def translation_value_validator(value: Any) -> str:
     """Validate that the value is a valid translation.
 
     - prevents string with HTML
+    - prevents strings with single quoted placeholders
     - prevents combined translations
     """
     value = cv.string_with_no_html(value)
+    value = cv.string_no_single_quoted_placeholders(value)
```

Comment:
`cv.string_no_single_quoted_placeholders(value)` returns a `str` like `cv.string_with_no_html(value)`, can we by not redefining `value` avoid the `str` convertion at return?

Reply:
I followed the same pattern as in `cv.string_with_no_html`. Should this be changed as well?

Reply:
`cv.string_with_no_html` can be done in a separate PR.

Reply:
I agree. I'll move it!